packages/numeric-input/src/presenters/stylesheet.js

```diff
@@ -0,0 +1,67 @@
+function getWrapperRulesByVariant(variant, props, themeData) {
+  switch (variant) {
+    case "line":
+      return {
+        width: "10px",
+        right: "0px",
+        bottom: "22%"
+      };
+    case "box":
+      return {
+        width: "36px",
+        right: "0px",
+        backgroundColor: themeData["input.box.default.backgroundColor"],
+        borderStyle: "solid",
+        borderColor: themeData["input.borderColor"],
+        borderWidth: themeData["basics.borderWidths.small"],
+        // height: "100%",
+        borderBottomColor: "transparent",
+        top: "0px",
+        bottom: "1px",
+        textAlign: "-webkit-center"
+      };
+    default:
+      return {};
+  }
+}
+function getSpinnerRulesByVariant(variant, themeData) {
+  switch (variant) {
+    case "line":
+      return {};
+    case "box":
+      return {};
+    default:
+      return {};
+  }
+}
+
+export default function stylesheet(props, themeData) {
+  const { disabled, hasFocus, hasHover, isPressed, variant } = props;
+
+  return {
+    spinnerWrapper: {
+      boxSizing: `border-box`,
+      position: `absolute`,
+      zIndex: 1,
+      ...getWrapperRulesByVariant(variant, props, themeData)
+    },
+    spinner: {
+      display: "block",
+      "svg *": {
+        fill: themeData["input.indicator.default"]
+      },
+
+      "&:active svg *": {
+        fill: themeData["colorScheme.indicator.pressed"]
+      }
+    },
+    iconUp: {
```

Comment:
If this is for centering of the arrows, having hard-coded padding may lead to more conditionals when you switch densities as well. I think the alignment is usually centered so something like `flex` that helps flex the space is a good idea, but you're flexing the svgs, when you might want to instead have a flex container, then just `justify-content`, `align-items`, `align-content` to move the elements within the container.